_posts/2021-08-02-how-to-update-to-terraform-1-x.adoc

```diff
@@ -0,0 +1,213 @@
+---
+title: How to update to Terraform 1.x
+categories: Upgrades
+image: /assets/img/guides/refresh_icon.png
+excerpt: Learn how to update your infrastructure code that depends on the Gruntwork Infrastructure as Code Library to Terraform 1.x.
+tags: ["aws", "terraform", "terragrunt"]
+cloud: ["aws"]
+redirect_from: /static/guides/upgrades/how-to-update-to-terraform-1-x/
+---
+:page-type: guide
+:page-layout: post
+
+:toc:
+:toc-placement!:
+
+// GitHub specific settings. See https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74 for details.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+toc::[]
+endif::[]
+
+== Intro
+
+This guide will walk you through how to update any code that depends on the
+https://gruntwork.io/infrastructure-as-code-library/[Gruntwork Infrastructure as Code Library] to
+https://www.terraform.io/upgrade-guides/1-0.html[Terraform 1.x]. Terraform v1.0
+is an unusual release in that its primary focus is on stability. Terraform
+v1.0.0 intentionally has no significant changes compared to Terraform v0.15.5.
+You can consider the v1.0 series as a direct continuation of the v0.15 series.
+=== What you'll learn in this guide
```

Comment:
You need a new line before this. Otherwise, it oddly wraps into the previous line:

![Screen Shot 2021-09-06 at 12 20 50 PM](https://user-images.githubusercontent.com/711908/132209779-28b0f344-4ada-445f-9700-da98240399d8.png)

Reply:
Good catch. Fixed in 174a3e3.